tahoma_api/client.py

```diff
@@ -97,7 +99,7 @@ async def fetch_event_listener(self, listener_id: str) -> List[Any]:
         Per-session rate-limit : 1 calls per 1 SECONDS period for this particular
         operation (polling)
         """
-        response = await self.__do_http_request("POST", f"events/{listener_id}/fetch")
+        response = await self.__post(f"events/{listener_id}/fetch")
```

Comment:
Could you add humps here as well?

Reply:
Why? We don't build the model here.

Reply:
Decimalising is not just for the models. Currently, no model has been created for the event listener, however this JSON can be consumed already and it would be good to have snake_case keys in there, instead of camelCase.

Otherwise, we add it back in a new PR, where we also add proper models.

Reply:
I prefer to do it in another PR when the model will be ready. Here it will make not yet sense.